docbook/reference/en/en-US/modules/Jaxb.xml

```diff
@@ -471,7 +462,6 @@ public void newBook(@BadgerFish Book book) {...}
 
       </para>
       <para>
-
          The default Jettison Mapped Convention would return JSON that looked like this:
```

Comment:
This should go, right?

Reply:
See
```
<programlisting>
java.lang.IllegalStateException: Invalid JSON namespace: http://jboss.org/books
at org.codehaus.jettison.mapped.MappedNamespaceConvention.getJSONNamespace(MappedNamespaceConvention.java:151)
at org.codehaus.jettison.mapped.MappedNamespaceConvention.createKey(MappedNamespaceConvention.java:158)
at org.codehaus.jettison.mapped.MappedXMLStreamWriter.writeStartElement(MappedXMLStreamWriter.java:241)
</programlisting>
```

Reply:
yes, the @BadgerFish things are part of jettison providers, let me remove them.